bin/MadGraph5_aMCatNLO/cards/production/13TeV/WJets_HT_LO_MLM/WJetsToQQ_HT-600toInf/WJetsToLNu_HT-600toInf_proc_card.dat

```diff
@@ -0,0 +1,25 @@
+import model sm-ckm_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
+#import model sm-ckm_no_b_mass
+
+define p = p b b~
+define j = j b b~
+define q = j b b~
```

Comment:
yes, it does not hurt - will remove it for consistency though

> On 05 Mar 2015, at 14:29, Josh Bendavid <notifications@github.com> wrote:
> 
> In bin/MadGraph5_aMCatNLO/cards/production/13TeV/WJets_HT_LO_MLM/WJetsToQQ_HT-600toInf/WJetsToLNu_HT-600toInf_proc_card.dat <https://github.com/cms-sw/genproductions/pull/233#discussion_r25862808>:
> 
> > @@ -0,0 +1,25 @@
> > +import model sm-ckm_no_b_mass
> > +#switch to diagonal ckm matrix if needed for speed
> > +#import model sm-ckm_no_b_mass
> > +
> > +define p = p b b~
> > +define j = j b b~
> > +define q = j b b~
> This is confusing, since the b's are being added twice. I assume madgraph is smart enough to remove duplicates from the definition?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/cms-sw/genproductions/pull/233/files#r25862808>.
>